graphium/finetuning/utils.py

```diff
@@ -0,0 +1,27 @@
+from copy import deepcopy
+
+
+def modify_cfg_for_finetuning(cfg):
```

Comment:
This could be a function within `FeedForwardNN` and `FullGraphNetwork`

Reply:
I am not sure, we can remove the function and replace inside the networks. As discussed in that #411, it might be possible to have a similar function within load_architecture in `graphium/data/_loader.py` but depends on the PR.